summary.txt

```diff
@@ -0,0 +1,52 @@
+Results/observations for group00
+Christine Tseng, Zuzanna Balewski
+
+We developed two outlier detection methods (see README.md), which produced
+slightly different results. While individually the strategies may not be ideal,
+we believe volumes identified by both more likely to be true outliers (or at
+least less dependent on possible biases in our methods).
+
+It would be very time consuming and expensive to reject scans with any
+outlier volumes, so we suggest excluding scans with more than 2
+(non-consecutive) outlier volumes.
+
+For group00 data, our outlier detection identifies the following volumes:
+group00_sub02_run1.nii 1, 2, 3, 158
+group00_sub02_run2.nii 35
+group00_sub05_run1.nii 20, 21, 22, 23
+group00_sub06_run1.nii 1, 2
+group00_sub06_run2.nii 22, 23
+group00_sub08_run1.nii 50
+group00_sub08_run2.nii 105, 106, 108, 156, 157, 158
+
+Using our rejection criterion, the following scans should be excluded:
+
+-- group00_sub02_run1.nii: The outlier volumes fall near the beginning
```

Comment:
For this description, I was hoping for your assessment from looking at the volumes or the metrics, as to whether they subject had moved, or whether you see evthere was some other source for the differences you've detected.